### DISCUSSION TITLE
Is the compose example and documented env variables wrong?

### DISCUSSION THREAD

**Discussion by danielo515** [librenms/docker#373] [Q&A]
I tried the following command:
```
docker run -d -p 8000:8000 --name librenms \
  -v $(pwd)/data:/data \
  -e "DB_HOST=192.168.0.107" \
  -e "MYSQL_USER=user" \
  -e "MYSQL_PASSWORD=password" \
  librenms/librenms:latest
```
And I get this error: `ERROR: Either DB_PASSWORD or DB_PASSWORD_FILE must be defined`

Then I just changed to this:
```
docker run -d -p 8000:8000 --name librenms \
  -v $(pwd)/data:/data \
  -e "DB_HOST=192.168.0.107" \
  -e "MYSQL_USER=user" \
  -e "DB_PASSWORD=pass" \
  librenms/librenms:latest
```
And then at lest it tired to connect to the database. It is maybe the documentation wrong?

**Comment by danielo515** [comment]
I can confirm that using DB_PASSWORD and DB_USER everything works

**Comment by crazy-max** [comment]
@danielo515 `MYSQL_USER` / `MYSQL_PASSWORD` is for the MariaDB container not LibreNMS: https://github.com/librenms/docker/blob/master/doc/docker/environment-variables.md#database

**Comment by danielo515** [comment]
Then where are documented the DB_USER and DB_PASSWORD env variables?

**Comment by crazy-max** [comment]
@danielo515 See the link in my previous comment.

**Comment by danielo515** [comment]
> @danielo515 See the link in my previous comment.

I saw your link, and I may be blind, but I didn't saw a single reference about how to provide the database credentials to librenms
